lib/bank_scrap.rb

```diff
@@ -7,6 +7,7 @@
 require 'bank_scrap/bank'
 require 'bank_scrap/account'
 require 'bank_scrap/transaction'
+require 'bank_scrap/banks/openbank/openbank'
```

Comment:
Quizás algo así:

lib/bank_scrap/banks/banks.rb

module BankScrap
  module Banks
  end
end

lib/banck_scrap.rb

module BankScrap
  # autoload only requires the file when the specified
  # constant is used for the first time
  autoload :Bankinter,  'bank_scrap/banks/bankinter'
  autoload :Bbva,       'bank_scrap/banks/bbva'
  autoload :Ing,        'bank_scrap/banks/ing'
  Banks.autoload :Openbank, 'bank_scrap/banks/openbank/openbank'
  Banks.autoload :Santander, 'bank_scrap/banks/santander/santander'
end

Una discusión sobre autoload: https://bugs.ruby-lang.org/issues/5653